src/main/java/de/bonndan/nivio/input/LinksResolver.java

```diff
@@ -0,0 +1,81 @@
+package de.bonndan.nivio.input;
+
+import de.bonndan.nivio.ProcessingException;
+import de.bonndan.nivio.input.dto.ComponentDescription;
+import de.bonndan.nivio.input.dto.LandscapeDescription;
+import de.bonndan.nivio.input.external.LinkHandlerFactory;
+import de.bonndan.nivio.util.SafeAssign;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+
+/**
+ * Resolves all links of all landscape components.
+ */
+public class LinksResolver {
```

Comment:
Why does this not extend `Resolver`?

Reply:
You're right, I have changed it based on your previous comment.